packages/react-dom/src/__tests__/ReactServerRenderingHydration-test.js

```diff
@@ -9,10 +9,13 @@
 
 'use strict';
 
+global.TextEncoder = require('util').TextEncoder;
```

Comment:
Interestingly we introduced a breaking change which means that `react-dom/server` including the renderToString API now depends on TextEncoder (because we construct one on module init).

Reply:
What happens in the browser?

Reply:
`new TextEncoder()` - throws in IE.